### PR TITLE
xlint a few templates

### PR DESCRIPTION
**Pull request by svenper**

### CODE REVIEW

File: srcpkgs/kernel-libc-headers/template [hunk 1]
```diff
@@ -1,17 +1,17 @@
 # Template file for 'kernel-libc-headers'
 pkgname=kernel-libc-headers
 version=4.19.0
-revision=2
+revision=3
 bootstrap=yes
-nostrip=yes
```
Comment (CameronNemo) [inline srcpkgs/kernel-libc-headers/template]: leave this change out. somebody likely wants to keep the `git blame` history clean. only do short_desc or none at all.

File: srcpkgs/libogg/template [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
 pkgname=libogg
 version=1.3.3
-revision=1
+revision=2
 build_style=gnu-configure
-short_desc="The Ogg bitstream file format library"
+short_desc="Ogg bitstream file format library"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="BSD"
 homepage="http://www.xiph.org/ogg/"
```
Comment (CameronNemo) [inline srcpkgs/libogg/template]: Use HTTPS for the homepage/distfiles here.

**Comment by CameronNemo** [comment]
I would only bump revision if you are adding a LICENSE file (nice find on those). One helpful change would be using https whenever possible. For the homepage this is fairly simple to detect I would imagine.

**Comment by Duncaen** [comment]
We decided against doing bulk xlint changes all the last times this came up before.

**Pull request closed without merging**